frontend/src/app/containers/ExperimentPage.js

```diff
@@ -157,12 +157,13 @@ export class ExperimentDetail extends React.Component {
 
   renderLocaleWarning() {
     const { experiment, locale } = this.props;
-    if (experiment.locales && locale && !experiment.locales.includes(locale)) {
+    if (locale && ((experiment.locales && !experiment.locales.includes(locale)) || (experiment.locale_blocklist && experiment.locale_blocklist.includes(locale)))) {
```

Comment:
@6a68 maybe. But this PR is only rendering a warning message to users, so it's not excluding it from that user's list. Do you think it still makes sense for a selector here? Like `renderLocalWarningSelector`, then test to see if the experiment exists in that collection? Seems like a bit of overhead.